tests/src/core/getExports.js

```diff
@@ -307,4 +310,35 @@ describe('getExports', function () {
 
   })
 
+  context('alternate parsers', function () {
+    const configs = [
+      // ['string form', { 'typescript-eslint-parser': '.ts' }],
+      ['array form', { 'typescript-eslint-parser': ['.ts', '.tsx'] }],
```

Comment:
I started with just one string, then made it support both a string and an array.

Then I decided I was making it too complicated.

Do you think it would be better to support both?